lib/beaker/host/windows/exec.rb

```diff
@@ -108,4 +108,14 @@ def selinux_enabled?()
     false
   end
 
+  # Determine if cygwin is actually installed on the SUT. Differs from
+  # is_cygwin?, which is just a type check for a Windows::Host.
+  #
+  # @return [Boolean]
+  def cygwin_installed?
+    output = exec(Beaker::Command.new('cygcheck --check-setup cygwin'), :accept_all_exit_codes => true).stdout 
+    return true if output.match(/cygwin/) and output.match(/OK/)
```

Comment:
Based on your comment I think you meant to change `and` to `&&`?

Re: exit codes.  You checked multiple Cygwin versions and they all respond the same way? When the check fails what if it responds with the string `NOT OK`?  I just figured the exit code check would insulate you from such differences.  If the binary is not found it should yield `127` (command not found), else the value from `cygcheck`